packages/terminology/README.md

```diff
@@ -5,30 +5,34 @@
 This project provides a term server for lookups in SNOMED, ICD, LOINC, and RXNORM, CPT, and CVX and crosswalks between SNOMED and ICD.
 Currently, this is mostly intended for internal use.
 
-## Download Metathesaurus
-
-You can download the Metathesaurus from the following link:  
-[Metathesaurus Download](https://www.nlm.nih.gov/research/umls/licensedcontent/umlsknowledgesources.html)  
-**Note:** The file is large - 4 GB compressed and 28 GB uncompressed. Releases occur twice a year, so stay alert!
-
 ## Getting Started
 
-### Start the Term Server Locally
+### For Metriport Devs
 
 Set up the `.env` with the following variables:
 
 - AWS_REGION=<staging-region>
 - TERMINOLOGY_BUCKET=<umls-bucket>
 
-To download the terminology database to your local environment and initiate the service, run this command:
+These env vars will be used to download the UMLS metathesaurus file from S3 to your local directory
+
+### For everyone else
+
+Download the Metathesaurus from the following link:  
+[Metathesaurus Download](https://www.nlm.nih.gov/research/umls/licensedcontent/umlsknowledgesources.html)  
+**Note:** The file is large - 4 GB compressed and 28 GB uncompressed. Releases occur twice a year, so stay alert!
```

Comment:
ditto to what you said on slack - let's add a disclaimer at the top of the readme that "everyone else" will need to get a free UMLS license here: https://uts.nlm.nih.gov/uts/signup-login